test/connections/amendment.test.ts

```diff
@@ -0,0 +1,71 @@
+import { handleWsMessageLedgerEnableAmendments } from '../../src/connection-manager/wsHandling'
+import { destroy, query, setupTables } from '../../src/shared/database'
+import { LedgerResponseCorrected } from '../../src/shared/types'
+
+import ledgerResponseNoFlag from './fixtures/ledgerWithNoFlag.json'
+import ledgerResponseGotMajority from './fixtures/ledgerWithTfMajority.json'
+
+const flushPromises = async (): Promise<void> => new Promise(setImmediate)
+
+describe('Amendments', () => {
+  beforeAll(async () => {
+    await setupTables()
+  })
+
+  afterAll(async () => {
+    await destroy()
+  })
+
+  beforeEach(async () => {
+    await query('amendments_status').delete('*')
```

Comment:
Why are you deleting the amendments_status before and after each test?

Reply:
It's to clean up the amendments_status table in the db between each test otherwise the next test might have data from the previous one